scripts/rclone_update_nfo_titles.py

```
@@ -0,0 +1,596 @@
+# ---------------------------------------------------------------------------
+def run_rclone(
+    args: List[str],
+    *,
+    check: bool = True,
+    capture: bool = True,
+    input_text: Optional[str] = None,
+    timeout: Optional[float] = None,
+) -> subprocess.CompletedProcess:
+    cmd = ["rclone", *args]
+    LOG.debug("exec: %s", " ".join(cmd))
+    return subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)